draft-ietf-rtcweb-jsep.xml

```diff
@@ -2344,9 +2344,19 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
                         <t>The fields of the "o=" line MUST stay the
                         same except for the &lt;session-version&gt;
-                        field, which MUST increment if the session
-                        description changes in any way, including the
-                        addition of ICE candidates.</t>
+                        field, which MUST increment for each call to
+                        the createOffer() API. The only actions which
+                        causes the &lt;session-version&gt; to
+                        increment is a call to the createOffer()
+                        API. If the Javascript has provided an offer
+                        to a remote side that might be confused by
+                        receiving different SDP with the same
+                        &lt;session-version&gt;, it should ensure that
+                        the createOffer() API is used to create the
+                        SDP such that all &lt;session-version&gt; of
+                        the SDP that the remote side will see have a
+                        unique &lt;session-version&gt;.</t>
+
```

Comment:
Regrettably, this is not what we agreed:

"V is the version in the o=line in last call to setLocal
o=V+1 on CreateOffer() if if anything changes 
"